webapp/tests/karma/ts/providers/translate-utils.provider.spec.ts

```diff
@@ -0,0 +1,143 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+import * as MessageFormat from 'messageformat';
+
+import {
+  TranslateMessageFormatCompilerProvider,
+  TranslateParserProvider,
+} from '@mm-providers/translate-utils.provider';
+
+let provider;
```

Comment:
This variable can be inside the scope of `describe('TranslateUtils providers'`

Reply:
Sure, it can. Does it make a big difference?  :)